clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -155,9 +155,12 @@ void goo() {
   // expected-error@+1 {{'loop_coalesce' attribute requires a positive integral compile time constant expression}}
   [[intel::loop_coalesce(0)]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
-  // expected-error@+1 {{'max_interleaving' attribute requires a non-negative integral compile time constant expression}}
+  // expected-error@+1 {{'max_interleaving' attribute requires integer constant between 0 and 1 inclusive}}
```

Comment:
This is a bit strange since technically it can only be 0 and 1. I think we should create a new diagnostic saying `'max_interleaving' attribute requires integer constant value 0 or 1`

Reply:
Thanks @elizabethandrews for reviews and feedbacks. I thought about to create a new diagnostic then changed my mind due to https://github.com/intel/llvm/blob/d5e4379674fd0189780ff613f7bc1cebb6962ead/clang/lib/Sema/SemaDeclAttr.cpp#L7761